toolbox/strings.py

```diff
@@ -102,6 +110,76 @@ def is_invite(string: str, *, fullmatch: bool = True) -> bool:
     return False
 
 
+class MarkdownFormat(IntFlag):
```

Comment:
I think there should be docs for what enum is. (I dont know if our doc system allows you to do a docstring under each variable in a class. cc @HyperGH )

Reply:
Sphinx does, yeah.

Reply:
What @parafoxia said.

Reply:
Do I have to do something special for that or can I just write a docstring like I would for a function?

Reply:
Just like you would a function. Though you just need the description for each.

Reply:
1. Full stops
2. It may be better to word it in such a way that the use is described. For example, "Remove strikethroughs caused by 2 tildes."